README.md

```diff
@@ -1,10 +1,10 @@
-# NZCP.js
+# NZCP.js &emsp; ![npm](https://img.shields.io/npm/v/@vaxxnz/nzcp) ![NPM](https://img.shields.io/npm/l/@vaxxnz/nzcp) ![npm](https://img.shields.io/npm/dw/@vaxxnz/nzcp)
 
 A JavaScript implementation of [NZ COVID Pass](https://github.com/minhealthnz/nzcovidpass-spec) verification, New Zealand's proof of COVID-19 vaccination solution, written in TypeScript. All contributions welcome 🥳
 
 We also have a [Rust implementation](https://github.com/vaxxnz/nzcp-rust/) available.
 
-> This library can be used for both Client and Server-side implementations.
+> This library can be used for both in browser and node.
```

Comment:
tiny pedantic thing, but capitalise the N pls 😆

Reply:
@oeed done, any other notes?